packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -1,6 +1,10 @@
 import 'whatwg-fetch';
 import { CallApi } from '../types';
 
+const cacheAvailable = 'caches' in self;
+const NOT_MODIFIED = 304;
```

Comment:
response numbers should be in a separate `constants` file

Reply:
Thanks, will do!